packages/expo-screen-capture/src/ScreenCapture.ts

```diff
@@ -8,33 +8,29 @@ const emitter = new EventEmitter(ExpoScreenCapture);
 
 const onScreenshotEventName = 'onScreenshot';
 
+// @needsAudit
```

Comment:
What's involved in an audit? Does reviewing the method descriptions as part of this PR count as an audit? 😅

Reply:
This is the tag @brentvatne asked me to add to comment blocks which are exposed via API generation on the website. AFAIK there i an docs QA planned which, in which most of descriptions will be proof-read, corrected na approved by the other team members. Few more details about annotation can be found in ENG-745.

Reply:
I see.. well if we're adding it to all of them then that's fair enough, but imo after the changes in this PR there's nothing I would change about these docs 👍 not sure if that qualifies as an audit

Reply:
I think that audit in this case means also to adjust wording and apply other good practices or preferred inguistic forms listed in the API docs guide written by James:
* https://github.com/expo/expo/blob/master/guides/Writing%20API%20Documentation.md